Adding an option for using IPs instead of URLs

temores opened the pull request:
Some customers don't have the capability in their firewalls to add URL filters, in that case IP lists are needed, our Datacenter ip list should be used instead.

azurecla commented:
Hi __@4c74356b41__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

PRmerger commented:
@4c74356b41 : Thanks for your contribution to the Azure documentation! The author, @azdocprs, has been notified to review your proposed change.

KatieCumming commented:
@NarayanAnnamalai - Hi Narayan, please review this change. If it's okay, please update this note format to Markdown. Thanks! Katie
#hold-off

carmonmills commented:
I reviewed the change and it looks good.  

#sign-off

PRmerger commented:
Only the author of the article can use the sign off comment to add the ready-to-merge label.

jomolnar commented:
@NarayanAnnamalai Hi Narayan. Please resolve the conflicts to enable a merge. Usually, authors of the files are able to fix merge conflicts using the process here on the [wiki](https://microsoft.sharepoint.com/teams/azurecontentguidance/wiki/Pages/Resolve a local merge conflict yourself.aspx). Thanks.

4c74356b41 commented:
Just a ping ;)

jimdial commented:
@4c74356b41 thanks for your contribution. Due to the merge conflict, the changes have been incorporated in this [PR](https://github.com/Azure/azure-content-pr/pull/32589/).

jimdial commented:
#please-close

temores committed:
Adding an option for using IPs instead of URLs

Some customers don't have the capability in their firewalls to add URL filters, in that case IP lists are needed, our Datacenter ip list should be used instead.
